createplaylist/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --primary-videoLib-color: #ff3b30;
+  --grey-background-card: #A8A8A8;
+  --body-background-color: #f7f7f7;
+}
+
+body {
+  margin-top: 5rem;
+  background-color: var(--body-background-color);
+}
+
+/** Navigation bar **/
+.nav-box-bar.desktop-nav {
+  position: fixed;
+  top: 0;
+  background-color: var(--color-white);
+  padding: 1rem 5rem;
+}
+
+.logo-text {
+  font-size: 1.5rem;
+  font-weight: 900;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.logo-text span {
+  color: var(--primary-videoLib-color);
+}
+
+#social-links-bar .no-bg.hov.btn-solid-primary {
+  height: auto;
+  width: auto;
+  background-color: var(--primary-videoLib-color);
+  color: var(--color-white);
+  margin: 0;
+  margin-right: 1rem;
+}
+
+#social-links-bar .no-bg.user-icon {
+  font-size: 1.6rem;
+  height: auto;
+}
+
+#social-links-bar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+}
+
+/** End of Navigation bar **/
+
+/** Login and Signup Section **/
+section {
+  height: 85vh;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+}
+
+.auth-box-login {
+  background-color: var(--color-white);
+  padding: 1rem 1.5rem 2.5rem 1.5rem;
+  height: auto;
+  width: 30%;
+  box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
```

Comment:
suggestion:you might use same shodow in other places so better to make this one also as avariable